perf_tests/green-3-20km/input_albany_Enthalpy_FiniteElementAssembly_SingleWorkset.yaml

```diff
@@ -68,7 +68,7 @@ ANONYMOUS:
 
 # Final solution comparison
   Regression For Response 0:
-    Test Value: -1.20352971037
+    Test Value: -1.24971227017
```

Comment:
This is actually a new case I added (no memoization finite element assembly, green-3-20km) since I removed the no memoization from 1-10km and 1-7km. The test value matches what's in `green-3-20km/input_albany_Enthalpy_FiniteElementAssembly_Memoization.yaml`.